x11-misc/sxcs/sxcs-1.1.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs edo shell-completion
```

Comment:
Sort inherits. Note that tools like `pkgcheck` should inform you about this.

Reply:
> Sort inherits.

Done.

> Note that tools like pkgcheck should inform you about this.

I ran `pkgcheck scan --commits HEAD` before opening PR, it didn't say anything about sorting inherits.

I *did* get a warning about "codeberg" being invalid for `remote-id` but it's been set to codeberg for a while now and I don't think it used to warn on it before, so I'm not sure what to do about it.

Reply:
> I ran `pkgcheck scan --commits HEAD` before opening PR, it didn't say anything about sorting inherits.

I think it used to do that, but I couldn't get it to spit out a warning either. Strange.

Thanks for the fixes.



> I _did_ get a warning about "codeberg" being invalid for `remote-id` but it's been set to codeberg for a while now and I don't think it used to warn on it before, so I'm not sure what to do about it.

I don't get such a warning. And in fact, 'codeberg' appears in https://gitweb.gentoo.org/data/dtd.git/tree/metadata.dtd#n59

Maybe some tooling is outdated on your end?

Reply:
> Maybe some tooling is outdated on your end?

Likely. I don't do system updates very often so my local pkgcheck isn't up to date. I'll get it sorted out.